packages/editor-ui/src/stores/ui.ts

```diff
@@ -65,6 +67,7 @@ export const useUIStore = defineStore(STORES.UI, {
 				open: false,
 				mode: '',
 				activeId: null,
+				showAuthOptions: false,
```

Comment:
Good catch on types here. Naming is a bit off here but it's the flag that signals whether this modal should show new auth UI (radio buttons, ...). It is based on the credentials that is is opened for, it they are required - new UI will show otherwise not.

Reply:
this one does not seem used.. `requiredCredentials` on the edit modal is getting used for that

Reply:
at least that's my understanding here

Reply:
Yep, cleared that a bit now.